duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -51,15 +55,21 @@ public Http(String inMethod, String inHost, String inUri) {
     this(inMethod, inHost, inUri, DEFAULT_TIMEOUT_SECS);
   }
 
+  public Http(String inMethod, String inHost, String inUri, int timeout) {
+      this(inMethod, inHost, inUri, timeout, null, null);
+  }
   /**
    * Http constructor.
    *
-   * @param inMethod    The method for the http request
-   * @param inHost      The api host provided by Duo and found in the Duo admin panel
-   * @param inUri       The endpoint for the request
-   * @param timeout     The timeout for the http request
+   * @param inMethod         The method for the http request
+   * @param inHost           The api host provided by Duo and found in the Duo admin panel
+   * @param inUri            The endpoint for the request
+   * @param timeout          The timeout for the http request
+   * @param hostnameVerifier the hostname verifier; may be null
+   * @param socketFactory    the socket factory; may be null
    */
-  public Http(String inMethod, String inHost, String inUri, int timeout) {
+  public Http(String inMethod, String inHost, String inUri, int timeout,
```

Comment:
I wonder if we should move to a Builder at this point?  It's probably overkill for this change, but it would set us up in a better position going forward.

Reply:
Yes that would be a good direction for future config changes.